input/postgres/relation_column_stats.go

```diff
@@ -15,14 +16,19 @@ SELECT schemaname, tablename, attname, inherited, null_frac, avg_width, n_distin
  WHERE schemaname NOT IN ('pg_catalog', 'information_schema')
 `
 
-func GetColumnStats(ctx context.Context, logger *util.Logger, db *sql.DB, globalCollectionOpts state.CollectionOpts, systemType string, dbName string) (state.PostgresColumnStatsMap, error) {
+func GetColumnStats(ctx context.Context, logger *util.Logger, db *sql.DB, globalCollectionOpts state.CollectionOpts, systemType string, dbName string, server *state.Server) (state.PostgresColumnStatsMap, error) {
 	var sourceTable string
 
 	if StatsHelperExists(ctx, db, "get_column_stats") {
 		logger.PrintVerbose("Found pganalyze.get_column_stats() stats helper")
 		sourceTable = "pganalyze.get_column_stats()"
+		server.SelfTest.MarkDbCollectionAspectOk(dbName, state.CollectionAspectColumnStats)
 	} else {
 		if systemType != "heroku" && !connectedAsSuperUser(ctx, db, systemType) && globalCollectionOpts.TestRun {
```

Comment:
ah found it, I am apparently using the superuser so it wasn't coming to this condition, therefore dbs that don't have the helper wasn't getting "OK".
same goes to column stats

Reply:
Ah cool, thanks!

Reply:
Ok, I pushed a fix. Could you try again? If you get any errors, can you share the full output (feel free to redact anything, obviously)?

Reply:
The latest version works well! 🎉